spec-source-orientation.html

```diff
@@ -474,20 +475,40 @@ <h3 id="deviceorientation"><span class=secno>4.1 </span>deviceorientation Event<
   that <code>alpha</code> is in the opposite sense to a compass heading. It also
   means that the angles do not match the roll-pitch-yaw convention used in
   vehicle dynamics.</p>
-
-  <p> Implementations that are unable to provide absolute values for the three
-  angles may instead provide values relative to some arbitrary orientation, as
-  this may still be of utility. In this case, the <code>absolute</code> property
-  must be set to <code>false</code>. Otherwise, the <code>absolute</code>
-  property must be set to <code>true</code>.</p>
+  
+  <p>The <code>deviceorientation</code> event tries to provide relative values for
+  the three angles, based on just the accelerometer and the gyroscope. The 
```

Comment:
Maybe add after 'the three angles', in brackets 'relative to some arbitrary orientation'. Or in a new sentence 'The angles are relative to some arbitrary orientation'. Otherwise it may not be clear what 'relative' means here.